client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -38,7 +34,7 @@ const DnsList = React.createClass( {
 				<DnsRecord
 					key={ index }
 					dnsRecord={ dnsRecord }
-					deleteDns={ this.deleteDns }
+					deleteDns={ this.handleDeleteDns }
```

Comment:
I think you should keep the former name for this method. That's the [convention](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#react-method-names) we've been following so far. Or maybe I'm missing something :).